ref/android-features.md

```diff
@@ -0,0 +1,21 @@
+Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
+
+| Feature           | Supported | Notes  |
+|--------------------|:-----------------:|
+| [OpenGL ES](https://source.android.com/devices/graphics/arch-egl-opengl) <= 3.2   |  ✓  | |
+| [Vulkan](https://source.android.com/devices/graphics/arch-vulkan) |  ✓  | |
+| [Camera](https://source.android.com/devices/camera) |  ✓  | |
+| [Sensors](https://source.android.com/devices/sensors) |  ✓  | Currently sensors a client device has are currently not connected to the Android container when streaming |
+| Location           |  ✓  |          |
+| NFC                |      |            |
+| [Bluetooth](https://source.android.com/devices/bluetooth) | | |
+| WiFi               |  ✓  | Only simulated WiFi is provided to the Android instance |
+| [Multi Display](https://source.android.com/devices/tech/display/multi_display) | | |
+| [Data Use](https://source.android.com/devices/tech/datausage)]| | |
+| Telephony / Mobile connectivity | | |
+| Hardware accelerated video decoding (H.264) | ✓ | |
+| Hardware accelerated video encoding (H.264) | | |
```

Comment:
```suggestion
| Hardware-accelerated video decoding (H.264) | ✓ | |
| Hardware-accelerated video encoding (H.264) | | |
```

Reply:
Why do we have two rows for this? Both supported and not supported? ;)

Reply:
One row for decoding, one for encoding. Encoding is currently not supported but decoding is.